docs/en/operation/Metrics-System.md

```diff
@@ -33,31 +34,46 @@ Each instance can report to zero or more sinks, found [here](https://github.com/
 
 ## Configuration
 
-The metrics system is configured via a configuration file that Alluxio expects to be present at
-`$ALLUXIO_HOME/conf/metrics.properties`. A custom file location can be specified via the
-`alluxio.metrics.conf.file` configuration property. Alluxio provides a `metrics.properties.template`
-under the `conf` directory which includes all configurable properties. By default, MetricsServlet
-is enabled in Alluxio master and workers. You can send an HTTP request to "`/metrics/json/`" to get a
-snapshot of all metrics in JSON format.
+The metrics system is configured via a configuration file that Alluxio expects to be present at `$ALLUXIO_HOME/conf/metrics.properties`. 
+A custom file location can be specified via the `alluxio.metrics.conf.file` configuration property. 
+Alluxio provides a `metrics.properties.template` under the `conf` directory which includes all configurable properties 
+and guidance of how to specify each property. 
 
+### Default HTTP JSON Sink
 
-For example, this command get the metrics in JSON format from the master process running locally:
+By default, `MetricsServlet` is enabled in Alluxio leading master and workers. 
+
+You can send an HTTP request to `/metrics/json/` of the Alluxio leading master to get a snapshot of all metrics in JSON format. 
+Metrics on the Alluxio leading master is a summary of the cluster-wide aggregated metrics.
```

Comment:
`also contains`? The master also has its own instance metrics it reports.

Reply:
Modify the docs to include instance metrics information in Alluxio leading master